src/tasks/sortedPowNumbers.js

```diff
@@ -0,0 +1,11 @@
+const sortPowNumbers = (arr) => {
+  const powArr = []
+  arr.forEach((element) => {
```

Comment:
Please replace forEach to map - https://javascript.info/array-methods#transform-an-array